app/models/patient.rb

```diff
@@ -106,6 +106,7 @@ class Patient < ApplicationRecord
 
   validates_with PrimaryContactValidator, on: %i[api import]
   validates_with RaceValidator, on: %i[api import]
+  validates_with InternationalPhoneNumberValidator, on: %i[api import]
```

Comment:
I think we should test this in `models/patient_test.rb`.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1069/commits/c44c099e7284dd1335eed9023d81f655f7a29685